### PR TITLE
docs: show step arrows 

### LINKED ISSUE #704: [Docs]: Add a property for show step arrows in currency and input widget
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Engineering Ticket Link

https://github.com/appsmithorg/appsmith/issues/18041

### Release Date

Jan 1st/2nd week

### Release Number

v1.8.14 or v1.9 

### First Draft

#705 

### Loom video

[_Video_](https://www.loom.com/share/83d05a6795d14aae92c9ff4e8bf82163)

### Discord/slack/intercom Link if needed

_No response_

### PRD

https://github.com/appsmithorg/appsmith/issues/18041#issuecomment-1328959200

### Test plan/cases

_No response_

### Use cases or user requests

_No response_

### PR DESCRIPTION
**Pull request by dilippitchika**
Fixes #704 

- The new property is to show/hide the increment arrows in widgets
- Added new properties in currency input and input widget
- JS controllable property

### CODE REVIEW

File: website/docs/reference/widgets/currency-input.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ These properties allow you to edit the Currency Input widget. All of these prope
 | **Placeholder**           | Sets the placeholder text within the input box. Use to show a hint or example value to the user.                                                                                                       |
 | **Tooltip**               | Sets a tooltip that appears when the user hovers over the widget with the mouse. Use this to provide hints or extra information to the user.                                                           |
 | **Required**              | Sets whether the checkbox is a mandatory field. When the checkbox is within a Form widget, that Form's submit button will be automatically disabled until the Checkbox is checked.                     |
+| **Show step arrows**              | Control's visibility of step arrows. When turned off: The step arrows to increment or decrement the values in the widget will be hidden.                                                                        |
```
Comment (danciaclara) [inline website/docs/reference/widgets/currency-input.md:27]:
```suggestion
| **Show step arrows**              | Controls visibility of step arrows. Turning this toggle off hides the step arrows to increment or decrement the values in the widget.                                                                        |
```

File: website/docs/reference/widgets/input.md [hunk 2]
```diff
@@ -24,6 +24,7 @@ These properties allow you to edit the Input widget. All of these properties are
 | **Error message**   | Sets the text of the error message to display if user input is considered invalid.                                                                                                                     |
 | **Placeholder**     | Sets the placeholder text within the input box. Use to show a hint or example value to the user.                                                                                                       |
 | **Tooltip**         | Sets a tooltip for the widget. You can add hints or extra information about the required input from the user.                                                                                          |
+| **Show step arrows**              | Control's visibility of step arrows for a number type input. When turned off: The step arrows to increment or decrement the values in the widget will be hidden.                                                                        |
```
Comment (danciaclara) [inline website/docs/reference/widgets/input.md:27]: same as on the suggested change in the Currency input widget. Also, this is valid only for the Number input type right? so we should mention that.

**Review by danciaclara — CHANGES_REQUESTED** [review]
@jnikhila can you also check if this passes the linter test? Thanks :)

**Comment by jnikhila** [comment]
@danciaclara - Linter issues are fixed and your suggested changes are merged too.

**Review by Pranay105 — APPROVED** [review]

**Pull request merged**